types/protoc-plugin/protoc-plugin-tests.ts

```diff
@@ -0,0 +1,26 @@
+import * as protocPlugin from 'protoc-plugin';
```

Comment:
It should be `import protocPlugin = require('protoc-plugin');` here: https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
Done.